trace-protobuf/build.rs

```diff
@@ -61,20 +60,60 @@ fn generate_protobuf() {
 
     config.type_attribute("Span", "#[derive(Deserialize, Serialize)]");
     config.field_attribute(
-        ".pb.Span",
-        "#[serde(default)] #[serde(deserialize_with = \"deserialize_null_into_default\")]",
+        ".pb.Span.service",
+        "#[serde(default)] #[serde(deserialize_with = \"crate::serializers::deserialize_null_into_default\")]",
+    );
+    config.field_attribute(
+        ".pb.Span.name",
+        "#[serde(default)] #[serde(deserialize_with = \"crate::serializers::deserialize_null_into_default\")]",
+    );
+    config.field_attribute(
+        ".pb.Span.resource",
+        "#[serde(default)] #[serde(deserialize_with = \"crate::serializers::deserialize_null_into_default\")]",
+    );
+    config.field_attribute(
+        ".pb.Span.traceID",
+        "#[serde(default)] #[serde(deserialize_with = \"crate::serializers::deserialize_null_into_default\")]",
+    );
+    config.field_attribute(
+        ".pb.Span.spanID",
+        "#[serde(default)] #[serde(deserialize_with = \"crate::serializers::deserialize_null_into_default\")]",
+    );
+    config.field_attribute(
+        ".pb.Span.parentID",
+        "#[serde(default)] #[serde(deserialize_with = \"crate::serializers::deserialize_null_into_default\")]",
+    );
+    config.field_attribute(
+        ".pb.Span.start",
+        "#[serde(default)] #[serde(deserialize_with = \"crate::serializers::deserialize_null_into_default\")]",
+    );
+    config.field_attribute(
+        ".pb.Span.duration",
+        "#[serde(default)] #[serde(deserialize_with = \"crate::serializers::deserialize_duration\")]",
+    );
+    config.field_attribute(
+        ".pb.Span.meta",
+        "#[serde(default)] #[serde(deserialize_with = \"crate::serializers::deserialize_null_into_default\")]",
+    );
+    config.field_attribute(
+        ".pb.Span.metrics",
+        "#[serde(default)] #[serde(deserialize_with = \"crate::serializers::deserialize_null_into_default\")]",
+    );
+    config.field_attribute(
+        ".pb.Span.type",
+        "#[serde(default)] #[serde(deserialize_with = \"crate::serializers::deserialize_null_into_default\")]",
```

Comment:
I know we are only targeting duration, does it make sense to then apply it to everything if we weren't doing it before?

Or is this mainly because we might reject when any of these fields are empty?